og_access/src/BundleFormAlter.php

```diff
@@ -60,7 +60,7 @@ public function formAlter(array &$form, FormStateInterface $form_state) {
     $form['og']['og_enable_access'] = [
       '#type' => 'checkbox',
       '#title' => t('Enable OG access control'),
-      '#default_value' => $this->hasAccessControl(),
+      '#default_value' => $this->bundle ? FALSE : $this->hasAccessControl(),
```

Comment:
`!$this->bundle`

Reply:
Whoops! Fixed. Simpler to flip the statements, so it's not a double negative.